config/menus/maps.cfg

```diff
@@ -1,210 +1,88 @@
-reset_map_ui = 0
-selected_map = 0
-selected_mutators = 0
-selected_mode = -1
-last_mode = -1
-last_mutators = 0
+// CC BY-SA 4-0 https://creativecommons.org/licenses/by-sa/4.0/legalcode
+// Iceflower iceflower@gmx.de
+// Modified by MoonPadUSer moonpaduser@mailbox.org
+
+
+truncate_string = [
+    if (!=s (limitstring $arg1 $arg2) $arg1) [
+        result (concatword (limitstring $arg1 (- $arg2 3)) "...")
+    ] [
+        result (limitstring $arg1 $arg2)
+    ]
+]
+
+if (getalias hidemappreviews) [] [ hidemappreviews = 0 ]
```

Comment:
This should be replaced with simply `hidemappreviews = 0`

The `setpersist` and `setcomplete` are fine as is, since this is an actual setting.
Consider adding a setdesc if you wish this to act similar to a normal c++ setting variable.

Reply:
ok, I'll add a desc